Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -196,6 +198,19 @@ public bool StartServer()
             if (runInBackground)
                 Application.runInBackground = true;
 
+            // set a fixed tick rate instead of updating as often as possible
+            // * if not in Editor (it doesn't work in the Editor)
+            // * if not in Host mode
+            // => set both Update and FixedUpdate tick rate.
+#if !UNITY_EDITOR
+            if (!NetworkClient.active)
```

Comment:
Can you change this to `if (!NetworkClient.active && serverTickRate > 0)`, so you can set it to 0/something negative to turn it off (zero/negative values don't make sense anyways)?

Reply:
could set it to 9999999 to turn it off too.

Reply:
That'd still set the max. fps though. 
I was more thinking of people (like me) having code with a few more conditions on the target fps.
Is it headless, what kind of map is it running (higher fps for more important combat things, low fps for trading towns, that kinda thing)